adapter/config/tpr/client.go

```diff
@@ -187,7 +188,7 @@ func (cl *Client) RegisterResources() error {
 		for _, kind := range kinds {
 			list := &ConfigList{}
 			err := cl.dynamic.Get().
-				Namespace(api.NamespaceAll).
+				Namespace(v1.NamespaceAll).
```

Comment:
k8s is in constant state of flux, so not much we can do about their code migrations.